src/me/ryanhamshire/AutomaticInventory/AutomaticInventory.java

```diff
@@ -417,8 +417,61 @@ public FakePlayerInteractEvent(Player player, Action rightClickBlock, ItemStack
         }
     }
 
+    /**
+     * Function to check if a chest would open based only on its block above
+     * @param aboveBlockID the block above the ctest
+     * @return whether or not the chest would not open
+     */
     static boolean preventsChestOpen(Material aboveBlockID)
     {
-        return aboveBlockID != Material.CHEST && aboveBlockID.isSolid();
```

Comment:
change from `isSolid` to `isOccluding` (and remove the exceptions below).

Reply:
Resolved in c09fcad

I also removed the `Material.CHEST` check, as it is now covered by `isOccluding`.